styleguide/components/31-molecules/wizard-steps/wizard-steps.config.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+module.exports = {
+  status: 'alpha',
+  preview: '@preview-description-list',
+  collated: 'true',
+  collator: function (markup, item) {
+    return `<!-- Start: @${item.handle} -->\n<dt>${item.name}</dt><dd>${markup}</dd>\n<!-- End: @${item.handle} -->\n`;
+  },
+  context: {
+    steps: [
+      {text: 'first step', url: '#'},
+      {text: 'second step', url: '#'},
+      {text: 'third step', url: '#'},
+      {text: 'fourth step with a long name', url: '#'},
+      {text: 'inactive fifth step'},
+      {text: 'inactive sixth step'},
+      {text: 'inactive seventh step'}
+    ],
+    active: 3
+  },
+  variants: [
+    {
+      name: 'default',
+      preview: '@preview'
+    },
+    {
+      name: 'first',
+      preview: '@preview',
+      context: {
+        steps: [
+          {text: 'first step', url: '#'},
+          {text: 'second step', url: '#'},
+          {text: 'third step', url: '#'},
+          {text: 'fourth step with a long name', url: '#'},
+          {text: 'inactive fifth step'},
+          {text: 'inactive sixth step'}
+        ],
```

Comment:
It's not necessary to define the `steps` over and over in each variant, since it's already in the global context (and will be inherited by the variants)